demisto_sdk/commands/generate_docs/generate_script_doc.py

```diff
@@ -139,9 +139,11 @@ def get_script_info(script_path):
     tags = ', '.join(map(str, tags))
 
     from_version = get_from_version(script_path)
-
-    res = [{'Name': 'Script Type', 'Description': script_type},
-           {'Name': 'Tags', 'Description': tags}]
+    res = []
+    if script_type:
+        res.append({'Name': 'Script Type', 'Description': script_type})
+    if tags:
+        res.append({'Name': 'Tags', 'Description': tags})
```

Comment:
Why are we adding the fix both here and in the generate_table_section function?

Reply:
Because first we don't want to add a row with no description to the table.
But then in case nothing was added to the table, we wouldn't want to create an empty table.

Reply:
👍